api/src/list/list.module.ts

```diff
@@ -9,18 +9,18 @@ import { ListDao } from './dao/list.dao';
 import { CommentDao } from './dao/comment.dao';
 import { UsersDao } from '../users/daos/user.dao';
 import { CommentsService } from './comment.service';
-import { MoviesModule } from '../movie/movie.module';
 
 @Module({
-  imports: [PrismaModule, NotificationModule, UsersModule, MoviesModule],
-  controllers: [ListsController],
+  imports: [PrismaModule, NotificationModule, UsersModule],
+  controllers: [ListController],
   providers: [
-    ListsService,
+    ListService,
     ListDao,
     UsersService,
     UsersDao,
```

Comment:
We don't need these. Importing the `UserModule` should be enough (as we do with the `NotificationModule`).